source/localizable/getting-started/quick-start.md

```diff
@@ -243,6 +243,50 @@ create a `programmers` route that shows a list of famous programmers. By
 re-using the `people-list` component, you can do it in almost no code at
 all.
 
+## User Events
+
+So far, your application is listing data, but there is no way for the user
+to interact with the information.
+In web applications, you often want to listen for user events like clicks or hovers.
```

Comment:
@locks I see the component tag `click` and `hover` as things to be covered in later parts of the guides?
Also, what is future best practice that we want to show since Glimmer has "tagless" components where the actions are completely declared in the template rather than the component.